das/worker.go

```diff
@@ -86,7 +86,7 @@ func (w *worker) run(ctx context.Context, timeout time.Duration, resultCh chan<-
 
 	if w.state.jobType != recentJob {
 		log.Infow(
-			"finished sampling headers",
+			"finished receiving headers",
```

Comment:
I think this would cause more confusion - "recency" in the context of DASer is related to whether or not a header coming in is a new networkHead (a header being gossiped). Headers that are !recent can still be sampled as long as they are > the sampling window cutoff.